basic/rules.cpp

```diff
@@ -236,6 +236,7 @@ static GameDefs g = {
 	0, // STRICT_GUARD
 	1, // OCEAN_GUARD
 	BattleLogLevel::NORMAL,	// BATTLE_LOG_LEVEL
+	false,	// WORLD_EVENTS
```

Comment:
can you please use 1/0 to follow convention

Reply:
using `int` where `bool` is expected is bad. How others would know that just 0 and 1 are accepted? Using wrong datatypes could lead to unexpected errors. All other places where int is used instead of boolean must be updated so that compiler can tell what is correct and what is not.